fusion/README.md

```diff
@@ -13,22 +13,22 @@ ansible-playbook playbook_name.yml
 In almost every playbook, there are 2 lines that require an environment variable
 
 ```
-app_id: "{{ ansible_env.API_CLIENT}}"
-key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+issuer_id: "{{ ansible_env.ISSUER_ID}}"
+private_key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
 ```
-In this case, the variables are: API_CLIENT and PRIV_KEY_FILE
+In this case, the variables are: ISSUER_ID and PRIV_KEY_FILE
 To set them, you can temporarily enable them with:
 ```
-export API_CLIENT='pure1:apikey:123456789'
+export ISSUER_ID='pure1:apikey:123456789'
 export PRIV_KEY_FILE='/home/user/key.pem'
```

Comment:
Please change this env variable name too, just to be consistent (nice to have thing).